src/VirtoCommerce.Platform.Web/Controllers/Api/ChangeLogController.cs

```diff
@@ -51,12 +52,39 @@ public ActionResult<LastModifiedResponse> GetLastModifiedDate([FromQuery] string
             var result = new LastModifiedResponse
             {
                 Scope = scope,
-                LastModifiedDate = _lastModifiedDateTime.LastModified.UtcDateTime
+                LastModifiedDate = _lastModifiedDateTime.GetLastModified().UtcDateTime
             };
 
             return Ok(result);
         }
 
+        [HttpPost]
+        [Route("~/api/changes/changedentities")]
```

Comment:
```suggestion
        [Route("~/api/changes/changed-entities")]
```
IMO, `kebab-case` is easier to read for endpoint URLs.